cca_example_test.go

```diff
@@ -0,0 +1,187 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package stat_test
+
+import (
+	"fmt"
+	"log"
+	"math"
+
+	"github.com/gonum/floats"
+	"github.com/gonum/matrix/mat64"
+	"github.com/gonum/stat"
+)
+
+// Helper for getting a View of a SymDense.
+type symView struct {
+	sym        *mat64.SymDense
+	i, j, r, c int
```

Comment:
Ok but this is a style thing I don't understand -- is it differentiating between different types of fields?